### PR TITLE
Add before_init and runner_image runtime fields to the Stack

### PR DESCRIPTION
**Pull request by b4k3r**

### CODE REVIEW

**Review by marcinwyszynski — APPROVED** [review]

File: spacelift/data_stack.go [hunk 1]
```diff
@@ -82,6 +88,11 @@ func dataStack() *schema.Resource {
 				Description: "Name of the repository, without the owner part",
 				Computed:    true,
 			},
+			"runner_image": {
+				Type:        schema.TypeString,
+				Description: "Name of the Docker image used to process the Run",
```
Comment (cube2222) [inline spacelift/data_stack.go:93]: "used to process Runs" there's no single Run.

File: spacelift/data_stack.go [hunk 2]
```diff
@@ -156,6 +173,12 @@ func dataStackRead(d *schema.ResourceData, meta interface{}) error {
 		d.Set("project_root", nil)
 	}
 
+	if stack.RunnerImage != nil {
+		d.Set("runner_image", *stack.RunnerImage)
```
Comment (cube2222) [inline spacelift/data_stack.go:177]: I'm obviously not sure, but wouldn't just d.Set("runner_image", stack.RunnerImage) without the if around it work?

**Review by cube2222 — APPROVED** [review]
With nits.

File: spacelift/data_stack.go (hunk 2 above)
Comment (b4k3r) [reply spacelift/data_stack.go]: You are right, I've applied it to other fields as well.

**Pull request merged**